addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1888,10 +1940,10 @@ MockServer.include({
 
             // notify update of last_interest_dt
             const now = datetime_to_str(new Date());
-            this._mockWrite('mail.channel',
+            this._mockWrite('mail.channel', [
```

Comment:
Why this change? Wrong syntax? Shouldn't it be done as a fix?

Reply:
Just spotted wrong syntax, I think I got buggy behaviour due to this typo.

Reply:
I think the diff is correct, but it should be done as fix outside of this PR?

2nd param of _mockWrite is supposed to be an array of args as you did.

Reply:
Made a 15.0 PR that fix this `_mockWrite` call: https://github.com/odoo/odoo/pull/88009